fe/fe-core/src/test/java/org/apache/doris/utframe/DemoMultiBackendsTest.java

```diff
@@ -201,9 +201,10 @@ public void testCreateDbAndTable() throws Exception {
         Assert.assertEquals(BackendsProcDir.TITLE_NAMES.size(), result.getColumnNames().size());
         Assert.assertEquals("{\"location\" : \"default\"}", result.getRows().get(0).get(20));
         Assert.assertEquals("{\"lastSuccessReportTabletsTime\":\"N/A\",\"lastStreamLoadTime\":-1,\"isQueryDisabled\":false,\"isLoadDisabled\":false}",
-                result.getRows().get(0).get(BackendsProcDir.TITLE_NAMES.size() - 3));
-        Assert.assertEquals("0", result.getRows().get(0).get(BackendsProcDir.TITLE_NAMES.size() - 2));
-        Assert.assertEquals(Tag.VALUE_MIX, result.getRows().get(0).get(BackendsProcDir.TITLE_NAMES.size() - 1));
+                result.getRows().get(0).get(BackendsProcDir.TITLE_NAMES.size() - 4));
+        Assert.assertEquals("0", result.getRows().get(0).get(BackendsProcDir.TITLE_NAMES.size() - 3));
+        Assert.assertEquals(Tag.VALUE_MIX, result.getRows().get(0).get(BackendsProcDir.TITLE_NAMES.size() - 2));
+        Assert.assertEquals("/home/doris.SSD:SSD; /home/doris.HDD:HDD; ", result.getRows().get(0).get(BackendsProcDir.TITLE_NAMES.size() - 1));
```

Comment:
`/home/doris` Whether depends on DORIS_HOME？

Reply:
it depends on be's conf: storage_root_path

> `/home/doris` Whether depends on DORIS_HOME？